src/CribblyBackend.DataAccess/Repositories/GameRepository.cs

```diff
@@ -85,9 +89,28 @@ public async Task Create(Game game)
                     new { TeamId = team.Id });
             }
         }
-        public void Update(Game game)
+        public async Task<Game> Update(Game game)
         {
-            throw new System.NotImplementedException();
+            await connection.ExecuteAsync(
+                @"
+                    UPDATE Games g
+                    SET
+                    g.GameRound = @GameRound,
+                    g.WinnerId = @WinnerId,
+                    g.ScoreDifference = @ScoreDifference
```

Comment:
Do we also need to update the 2 rows in the scores table?

Reply:
I think we would only need to update that row when posting a score. This endpoint is more about updating the game table itself. We can try to have it update the games and scores too if you want.

Reply:
🔮 🤔 maybe we need to have two separate endpoints: one to update a game (for any reason other than a score update), and one to post scores. I don't think we should update the game's score difference without updating the scores table, because then the two tables would be out of sync.